ui/admin/tests/acceptance/host-catalogs/create-test.js

```diff
@@ -64,26 +66,41 @@ module('Acceptance | host-catalogs | create', function (hooks) {
 
     urls.hostCatalog = `${urls.hostCatalogs}/${instances.hostCatalog.id}`;
     urls.newHostCatalog = `${urls.hostCatalogs}/new`;
+    urls.newStaticHostCatalog = `${urls.newHostCatalog}?type=static`;
+    urls.newDynamicHostCatalog = `${urls.newHostCatalog}?type=aws`;
     // Generate resource couner
     gethostCatalogCount = () =>
       this.server.schema.hostCatalogs.all().models.length;
 
     authenticateSession({});
   });
 
-  test('Users can create new host catalogs', async function (assert) {
+  test('Users can create new static host catalogs', async function (assert) {
     assert.expect(1);
     const count = gethostCatalogCount();
-    await visit(urls.newHostCatalog);
+    await visit(urls.newStaticHostCatalog);
     await fillIn('[name="name"]', 'random string');
+    await fillIn('[name="description"]', 'random string');
+    await fillIn('[name="static_type"]', 'static');
+    await click('[type="submit"]');
+    assert.equal(gethostCatalogCount(), count + 1);
+  });
+
+  test('Users can create new dynamic host catalogs', async function (assert) {
+    assert.expect(1);
+    const count = gethostCatalogCount();
+    await visit(urls.newDynamicHostCatalog);
+    await fillIn('[name="name"]', 'random string');
+    await fillIn('[name="description"]', 'random string');
+    await fillIn('[name="types"]', 'aws');
     await click('[type="submit"]');
     assert.equal(gethostCatalogCount(), count + 1);
   });
 
   test('Users can cancel create new host catalogs', async function (assert) {
```

Comment:
We changed the previous test (`Users can create new static host catalogs`) to specify `static`. Should we do the same with the rest of the test in this file? I mean specify if applies for `static`, `plugin` or both?

Reply:
updated! ty!